api/src/pcapi/routes/backoffice_v3/pro_users/blueprint.py

```diff
@@ -59,7 +62,9 @@ def get(user_id: int) -> utils.BackofficeResponse:
         form=form,
         dst=dst,
         empty_form=empty_forms.EmptyForm(),
+        can_delete_user=_user_can_be_deleted(user),
         **user_forms.get_toggle_suspension_args(user, user_forms.SuspensionUserType.PRO),
+        **_get_delete_kwargs(user),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏
`can_delete_user=_user_can_be_deleted(user),` fait doublon avec 
`"can_be_deleted": _user_can_be_deleted(user),` dans les kwargs

Reply:
en effet un reste du dev que j'ai raté quand je faisait le ménage a la fin :)